.jazzy.yaml

```diff
@@ -0,0 +1,72 @@
+module: Blueprint
+title: Blueprint
+github_url: https://github.com/square/Blueprint
+skip_undocumented: false
+theme: fullwidth
+# TODO: to enable this, first rename docs whose names conflict with symbols
```

Comment:
Is this a TODO for this PR or future work?

Reply:
Future work.